docs/components_and_bundles/components/Locale/basic_usage.rst

```diff
@@ -4,7 +4,25 @@ Basic Usage
 LocaleContext
 -------------
 
-...
+CompositeLocaleContext
+~~~~~~~~~~~~~~~~~~~~~~
+	. getLocaleCode() method, a composite of different contexts which are prioritized in it (the one with highest priority is used if it exists). Returns a string (exception: LocaleNotFoundException)
+	. _construct()
+	. addContext(LocaleContextInterface $localeContext, int $priority)
+
+ImmutableLocaleContext
+~~~~~~~~~~~~~~~~~~~~~~
+	. _construct(string $localeCode)
+	. string getLocaleCode(), see above for details
+
+LocaleContextInterface
+~~~~~~~~~~~~~~~~~~~~~~
+	. string getLocaleCode(), see above for details
+
+ProviderBasedLocaleContext
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+	. _construct(LocalProviderInterface $localeProvider)
+	. string getLocaleCode(), see above for details
```

Comment:
Instead of these three paragraphs you have above I would move it under the `LocaleContext` subtitle like that:

```
LocaleContext
-------------

In the Locale component there are three LocaleContexts defined:
* `CompositeLocaleContext`
* `ImmutableLocaleContext`
* `ProviderBasedLocaleContext`
```